### ISSUE TITLE
hashi-up unable to ssh to fresh DigitalOcean droplet

### ISSUE THREAD

**Issue by coreycarvalho** [jsiebens/hashi-up#35]
When trying to run `hash-up nomad install` on a fresh digitalocean droplet, I'm met with the following error:
```bash
$ hashi-up nomad install --ssh-target-addr 67.207.94.29  --ssh-target-key ~/.ssh/id_rsa

There was an issue connecting to your target host.
This could happen when hashi-up can not reach the target host or when the private key authentication is invalid.

Reason: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain

How to fix this?

- check if the target host is reachable and an SSH server is running
- check if the user and the private key are valid
```

What's odd is that I am able to ssh onto the host using root without a problem. Below I'll outline additional context along with my machine specs and stuff.

**Context**
I'm provisioning my droplet with terraform, using the following config...
```bash
resource "digitalocean_droplet" "nomad_server" {
  image    = "ubuntu-22-04-x64"
  name     = "nomad-server"
  region   = "nyc1"
  size     = "s-1vcpu-1gb"
  ssh_keys = [data.digitalocean_ssh_key.corey_mb_air.id]
}
```
The ssh key above is the public key on my M1 MacBook Air running MacOS 12.5.1. When running `ssh root@67.207.94.29`, I am able to log onto the host without a problem. Passwordless login is also enabled.

**Comment by coreycarvalho** [comment]
Interesting.... I just tried this with the image `ubuntu-18-04-x64` and it worked completely fine. This only seems to break with image `ubuntu-22-04-x64`

**Comment by coreycarvalho** [comment]
Found the solution.... https://askubuntu.com/questions/1409105/ubuntu-22-04-ssh-the-rsa-key-isnt-working-since-upgrading-from-20-04

tl;dr: RSA SHA-1 hash algorithm is being deprecated. Using a more secure key allowed this to work 👍
